cards/nfts/nfts.go

```diff
@@ -24,6 +24,8 @@ type DB interface {
 	Create(ctx context.Context, nft NFT) error
 	// Get returns nft by token id and chain from database.
 	Get(ctx context.Context, tokenID int64, chain evmsignature.Chain) (NFT, error)
+	// GetNFTByID returns nft by id from database.
```

Comment:
maybe it's better to call this method GetNFTByCardID(ctx context.Context, cardID uuid.UUID)